evap/evaluation/tests/tools.py

```diff
@@ -53,6 +54,19 @@ def get_submit_assert_200(self, url, user):
         self.assertEqual(response.status_code, 200, 'url "{}" failed with user "{}"'.format(url, user))
         return response
 
+    def let_user_vote_for_course(self, user, course):
+        url = '/student/vote/{}'.format(course.id)
+        page = self.get_assert_200(url, user=user)
+        form = page.forms["student-vote-form"]
+        for contribution in course.contributions.all():
```

Comment:
please add a `.prefetch_related("questionnaires", "questionnaires__question_set")` for likely no performance gain but still.